src/component/Switch.jsx

```diff
@@ -0,0 +1,329 @@
+import React from "react";
+import styled from "styled-components";
+
+const Switch = () => {
+  const setDarkMode = () => {
+    document.querySelector("body").setAttribute("data-theme", "dark");
+    localStorage.setItem("selectedTheme", "dark");
+  };
+  const setlightMode = () => {
+    document.querySelector("body").setAttribute("data-theme", "light");
+    localStorage.setItem("selectedTheme", "light");
+  };
+
+  const selectedTheme = localStorage.getItem("selectedTheme");
+
+  if (selectedTheme === "dark") {
+    setDarkMode();
+  }
+
+  const toggleTheme = (e) => {
+    if (e.target.checked) setDarkMode();
+    else setlightMode();
+  };
+  return (
+    <StyledWrapper>
+      <label className="theme-switch">
+        <input
+          type="checkbox"
+          className="theme-switch__checkbox"
+          onChange={toggleTheme}
+          defaultChecked={selectedTheme === "dark"}
+        />
+        <div className="theme-switch__container">
+          <div className="theme-switch__clouds" />
+          <div className="theme-switch__stars-container">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 144 55"
+              fill="none"
+            >
+              <path
```

Comment:
Sve slike drzi u assets/images folderu. Ukoliko je slucaj kao ovde a to je da nemas png, jpg sliku, onda svg kao sto je ovde slucaj mozes da drzis npr unutar components/images. To ce biti react komponente koje ce vracati samo svg nazad